### ISSUE TITLE
Even after running Office.initialize(), Word object is undefined

### ISSUE THREAD

**Issue by nedbalzer** [OfficeDev/office-js#571]
## Expected Behavior
Word Add-in worked in Word 2016 but does not work with Penn State University's Office365. If I install the Word Web Add-in demo from https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/word-quickstart?tabs=visual-studio and substitute the code from my add-in into html and javascript files and adjust the manifest.xml, this code *does* run in Visual Studio 2017 -- that seems to launch word in a VS "sandbox"; it is also possible that 1) Penn State's installation of Office365 is hobbled and does not permit this add-in to run for some reason, or 2) Penn State's use of Cosign SSO with 2-factor authentication is breaking something.

## Current Behavior
Office.initialize indicates that Office is loaded (and the Office object is not null), but the Word object is null. Initial error seems to occur at word-win32-16.01.js (line 24, column 346444): Unable to get property 'getItem' of undefined or null reference.

I have also tried modifying the main.js initial call of Office.initialize to use Office.onReady() instead, but still get the same error.

To contact me I am at emb23@psu.edu

## Steps to Reproduce, or Live Example
  1. Extract add-in code (attached) to C:\PSUIndexer\
  2. Add \\localhost\C$\PSUIndexer as a trusted Add-in catalog
  3. Launch the Add-in in MS Word

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: Office 365; Word Version 1906 (Build 11727.20104 Click-to-Run)
* Operating System: Windows 10
[PSUIndexer.zip](https://github.com/OfficeDev/office-js/files/3321569/PSUIndexer.zip)

**Comment by kbrandl** [comment]
@nedbalzer sorry to hear that you've run into this issue; we'll investigate.

@weshi FYI, assigning this issue to you for investigation.

**Comment by weshi** [comment]
@kbrandl This is not in my area. Please help to reassign. Thanks.

**Comment by kbrandl** [comment]
@nedbalzer apologies for the lack of engagement on this issue. Are you still having this problem?

**Comment by nedbalzer** [comment]
I worked closely with Microsoft Office add-in support on this. For reasons I don’t understand, our old procedure of distributing the add-in as a download and installing it locally stopped working when our organization switched from local installs of Word 2016 to Office365 (also Word 2016). We were able to fix this by moving the add-in code (html, css and javascript) to a website and having the manifest.xml file point to it there.

So we no longer need you to look at this issue.


Thanks very much.

-- Ned Balzer (emb23@psu.edu)
